[2022-12-23T21:31:09.360+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: parking_pipeline.create_table scheduled__2022-12-23T21:29:06.490597+00:00 [queued]>
[2022-12-23T21:31:09.369+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: parking_pipeline.create_table scheduled__2022-12-23T21:29:06.490597+00:00 [queued]>
[2022-12-23T21:31:09.369+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-23T21:31:09.370+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2022-12-23T21:31:09.371+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-23T21:31:09.380+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_table> on 2022-12-23 21:29:06.490597+00:00
[2022-12-23T21:31:09.386+0000] {standard_task_runner.py:55} INFO - Started process 10858 to run task
[2022-12-23T21:31:09.390+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'parking_pipeline', 'create_table', 'scheduled__2022-12-23T21:29:06.490597+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/parking_dag.py', '--cfg-path', '/tmp/tmppupah9b_']
[2022-12-23T21:31:09.393+0000] {standard_task_runner.py:83} INFO - Job 32: Subtask create_table
[2022-12-23T21:31:09.589+0000] {task_command.py:389} INFO - Running <TaskInstance: parking_pipeline.create_table scheduled__2022-12-23T21:29:06.490597+00:00 [running]> on host 0330eddd684a
[2022-12-23T21:31:09.646+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Airflow
AIRFLOW_CTX_DAG_ID=parking_pipeline
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-23T21:29:06.490597+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-23T21:29:06.490597+00:00
[2022-12-23T21:31:09.647+0000] {sql.py:253} INFO - Executing: 
            drop table if exists rates;
            create table parking_data(
                rate float not null,
                symbol text not null
            );
        
[2022-12-23T21:31:09.658+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2022-12-23T21:31:09.795+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2022-12-23T21:31:09.801+0000] {sql.py:364} INFO - Running statement: 
            drop table if exists rates;
            create table parking_data(
                rate float not null,
                symbol text not null
            );
        , parameters: None
[2022-12-23T21:31:09.803+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 261, in execute
    return_last=self.return_last,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 338, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 369, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.DuplicateTable: relation "parking_data" already exists

[2022-12-23T21:31:09.812+0000] {taskinstance.py:1327} INFO - Marking task as UP_FOR_RETRY. dag_id=parking_pipeline, task_id=create_table, execution_date=20221223T212906, start_date=20221223T213109, end_date=20221223T213109
[2022-12-23T21:31:09.821+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 32 for task create_table (relation "parking_data" already exists
; 10858)
[2022-12-23T21:31:09.853+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-23T21:31:09.871+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
